server/channels/app/export_test.go

```diff
@@ -943,6 +943,54 @@ func TestExportDeletedTeams(t *testing.T) {
 	}
 }
 
+func TestExportSingleTeam(t *testing.T) {
```

Comment:
Can we also add tests where we can get errors or expect errors. Also it may be good to evaluate there is no DM/GMs exported.

Reply:
Done.